lib/web_ui/lib/src/engine/html/bitmap_canvas.dart

```diff
@@ -555,7 +559,8 @@ class BitmapCanvas extends EngineCanvas {
                 pathAsLine.left, pathAsLine.top, 1, pathAsLine.height);
 
         final html.HtmlElement element = buildDrawRectElement(
-            rect, paint, 'draw-rect', _canvasPool.currentTransform);
+            rect, paint, 'draw-rect', _canvasPool.currentTransform) as
+            html.HtmlElement;
```

Comment:
nit: these `html.HtmlElement` look strange when separated from the `as`. Do they need a new line?